.github/workflows/build_branch34.yml

```diff
@@ -37,7 +37,8 @@ jobs:
       envs: >-
         {
           "SCALA_PROFILE": "scala2.13",
-          "PYTHON_TO_TEST": ""
+          "PYTHON_TO_TEST": "",
+          "ORACLE_DOCKER_IMAGE_NAME": "gvenzl/oracle-xe:21.3.0"
```

Comment:
I tested it in my own repository, the environment variable did not overwrite successfully, and I haven't found the reason yet.